stubs/protobuf/google/protobuf/internal/containers.pyi

```diff
@@ -25,7 +26,7 @@ class BaseContainer(Sequence[_T]):
     def __getitem__(self, key: slice) -> list[_T]: ...
 
 class RepeatedScalarFieldContainer(BaseContainer[_ScalarV]):
-    def __init__(self, message_listener: MessageListener, message_descriptor: Descriptor) -> None: ...
+    def __init__(self, message_listener: MessageListener, type_checker: TypeChecker) -> None: ...
```

Comment:
The docstring says: `A type_checkers.ValueChecker instance`?

Reply:
Interestingly that's not actually a type that exist.
At runtime, they look like this:
https://github.com/protocolbuffers/protobuf/blob/master/python/google/protobuf/internal/type_checkers.py#L147

What we want is some kind of protocol for ducktyping things that have `CheckValue` and `DefaultValue`

Thanks for catching!